src/SOS/SOS.UnitTests/SOSRunner.cs

```diff
@@ -556,9 +556,10 @@ public static async Task<SOSRunner> StartDebugger(TestInformation information, D
                     // Turn on source/line numbers
                     initialCommands.Add(".lines");
 
-                    string setHostRuntime = config.SetHostRuntime();
-                    bool secureLoadDotNetExtensions = !(config.PrivateBuildTesting() || (setHostRuntime != null && setHostRuntime == "-none"));
-                    initialCommands.Add($"dx @Debugger.Settings.EngineInitialization.SecureLoadDotNetExtensions={(secureLoadDotNetExtensions ? "true" : "false")}");
+                    bool shouldVerifyDacSignature = !config.PrivateBuildTesting()
+                                                    && !"nightly".Equals(config.GetValue("BuildType"), StringComparison.OrdinalIgnoreCase)
```

Comment:
You should add an TestConfiguration extension method for BuildType at the end of this file.